container/container.go

```diff
@@ -45,6 +45,44 @@ const (
 	attributeTimestamp = "Timestamp"
 )
 
+// CopyTo writes deep copy of the [Container] to dst.
+func (x Container) CopyTo(dst *Container) {
+	if dst == nil {
+		return
+	}
+
+	dst.SetBasicACL(x.BasicACL())
+	dst.SetOwner(x.Owner())
+
+	var ver refs.Version
+	ver.SetMajor(x.v2.GetVersion().GetMajor())
+	ver.SetMinor(x.v2.GetVersion().GetMinor())
+	dst.v2.SetVersion(&ver)
+
+	dst.v2.SetNonce(nil)
```

Comment:
useless if nonce is set, can be done cleaner via variable `nonce`, same for attributes